packages/invoices/src/OutboundShipment/DetailView/reducer.test.ts

```diff
@@ -2,13 +2,14 @@ import {
   DocumentAction,
   Invoice,
   createColumns,
+  DocumentActionSet,
 } from '@openmsupply-client/common';
 import { placeholderInvoice } from './index';
 import { reducer, OutboundShipmentStateShape, OutboundAction } from './reducer';
-import { InvoiceLineRow } from './types';
+import { OutboundShipmentRow, OutboundShipmentAction } from './types';
```

Comment:
quite a few test updates to try make them easier to write as I can see quite a few being written!